client/package.json

```diff
@@ -14,11 +15,14 @@
     "@types/react-router-dom": "^5.1.8",
     "axios": "^0.23.0",
     "react": "^17.0.2",
+    "react-audio-player": "^0.17.0",
```

Comment:
Please use `react-media-player` instead of this.

Reply:
react-media-player isn't supporting react 17 version.